client/src/lib/components/DatasetListItem/DatasetListItem.svelte

```diff
@@ -16,104 +16,118 @@
   };
 </script>
 
-<li class="fr-p-3w">
-  <p class="fr-logo fr-logo--sm fr-p-0" title="république française">
-    {@html "Ministère<br />de la culture"}
-  </p>
-
-  <div class="fr-container">
-    <p class="fr-m-0">
-      {#if dataset.headlines}
-        <strong data-testid="headlines-title">
-          {@html dataset.headlines.title}
-        </strong>
-      {:else}
-        <strong>
-          {dataset.title}
-        </strong>
-      {/if}
+<li class="fr-p-3w container">
+  <a
+    data-test-id={`${dataset.title}`}
+    href={paths.datasetDetail({ id: dataset.id })}
+    title="Consulter cette fiche de données"
+  >
+    <p class="fr-logo fr-logo--sm fr-p-0" title="république française">
+      {@html "Ministère<br />de la culture"}
     </p>
 
-    <p class="fr-m-0 fr-text--sm fr-text-mention--grey">
-      {dataset.service}
-    </p>
+    <div class="container">
+      <p class="fr-m-0">
+        {#if dataset.headlines}
+          <strong data-testid="headlines-title">
+            {@html dataset.headlines.title}
+          </strong>
+        {:else}
+          <strong>
+            {dataset.title}
+          </strong>
+        {/if}
+      </p>
 
-    <div class="metadata-items fr-mt-1w">
-      <div>
-        <span class="fr-fi fr-fi-x-map-2-line" aria-hidden="true" />
-        <p class="fr-text--xs fr-my-0">
-          <span class="fr-text-mention--grey">Couverture géographique</span>
-          <br />
-          <span
-            >{GEOGRAPHICAL_COVERAGE_LABELS[dataset.geographicalCoverage]}</span
+      <p class="fr-m-0 fr-text--sm fr-text-mention--grey">
+        {dataset.service}
+      </p>
+
+      <div class="metadata-items fr-mt-1w">
+        <div>
+          <span class="fr-fi fr-fi-x-map-2-line" aria-hidden="true" />
+          <p
+            class="fr-text--xs fr-my-0 fr-px-1w metadata-geographical-coverage"
```

Comment:
Cette classe CSS ne semble pas définie

Par ailleurs on ajoute ici `fr-px-1w` sans l'ajouter à l'item "formats". Je crois qu'il faudrait le mettre au 2, ou aucun ?

```suggestion
            class="fr-text--xs fr-my-0 fr-px-1w"
```